tests/run.sh

```diff
@@ -4,6 +4,14 @@ set -ueo pipefail
 # Top-level test runner. Usage is "run.sh" to run tests in compile-only mode,
 # or "run.sh <runwasi>" where <runwasi> is a WASI-capable runtime to run the
 # tests in full compile and execute mode.
+#
+# By default this script will look for `clang` and `clang++` in $PATH and
+# assume that they are correctly configured with the sysroot in the default
+# location.  Alternatively, exporting $CC and $CXX allow more flexibility. e.g:
+#
+#  export CXX="<wasi-sdk>/bin/clang++ --sysroot <wasi-sdk>/share/wasi-sysroot"
+#  export CCC="<wasi-sdk>/bin/clang --sysroot <wasi-sdk>/share/wasi-sysroot"
```

Comment:
This should be `CC`?

Reply:
oops .. thanks.  I'd have some more changes to the test suite coming soon so I'll add that typo fix there.